src/music.ts

```diff
@@ -202,11 +202,11 @@ export class Music {
 
   getNoteParts(noteString: string): NoteParts {
     if (!noteString || noteString.length < 1) {
-      throw new Vex.RERR('BadArguments', 'Invalid note name: ' + noteString);
+      throw new Vex.RERR('BadArguments', `Invalid note name: ${noteString}`);
```

Comment:
How does this relate to babel removal? This PR is too many things

Reply:
All the changes in Music relate to Airbnb. They are done automatically by eslint.

Reply:
eslint fixes are okay with me -- I've rarely found them to break things.